src/storage/v3/conversions.hpp

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "storage/v3/property_value.hpp"
+
+#pragma once
+
+namespace memgraph::storage::v3 {
+
+template <typename TypedValue>
+TypedValue PropertyToTypedValue(const PropertyValue &value) {
```

Comment:
Why now move this to the `src/glue` part?

Reply:
Because these will only be used by the storage engine. Currently, we link `storage` to `query engine` but in the future, those two will be completely separate and the query engine will work with `TypedValue` and `Transport::Request/Response`

Reply:
I left these because I wanted to make sure that `query::v2` compiles with the changes I made